content/webapp/utils/digital-guides.ts

```diff
@@ -74,9 +74,8 @@ export const getGuidesRedirections = (
 ) => {
   const { req, res, resolvedUrl } = context;
   const { id: guideId, stopNumber, stopId, type, usingQRCode } = context.query;
-  const hasEgWorkCookie = getCookie('toggle_egWork', { req, res }) === 'true';
 
-  if (!usingQRCode) return;
+  if (toMaybeString(usingQRCode) !== 'true') return;
```

Comment:
Have approved as this works, but why is this change necessary?

Reply:
Ah yes sorry I forgot to explain this one!

We were just checking if `usingQRCode` was a query param, but not its actual value. So it could be `usingQRCode=dsjbsufjdjsofsuho` and still work. Which I guess is fine because no one inputs that manually anyway, but thought I'd make it have a clearer intention.

`isMaybeString` turns `string | string[] | undefined` into `string | undefined` (picking the first instance if an array.